_sass/page.scss

```diff
@@ -300,3 +300,24 @@ hr {
     } 
  }
 
+ /* width */
+::-webkit-scrollbar {
+    width: 15px;
+  }
+  
+  /* Track */
+  ::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px rgb(179, 179, 179); 
+    border-radius: 5px;
+  }
+   
+  /* Handle */
+  ::-webkit-scrollbar-thumb {
+    background: rgb(77, 50, 226); 
```

Comment:
The colour I don't feel goes with our theme. Maybe use some type of grey or the MLH blue theme colour